backend/src/schema/resolvers/reports.spec.js

```diff
@@ -139,55 +198,120 @@ describe('file a report on a resource', () => {
           it('which belongs to resource', async () => {
             await expect(
               mutate({
-                mutation: reportMutation,
+                mutation: fileReportMutation,
                 variables: { ...variables, resourceId: 'abusive-user-id' },
               }),
             ).resolves.toMatchObject({
               data: {
                 fileReport: {
-                  id: expect.any(String),
+                  reportId: expect.any(String),
+                  resource: {
+                    name: 'abusive-user',
+                  },
                 },
               },
               errors: undefined,
             })
           })
 
-          it('creates only one report for multiple reports on the same resource', async () => {
+          it('only one report for multiple reports on the same resource', async () => {
```

Comment:
You could end the sentence in 3rd person English in the `it`. E.g. `it avoids duplicate reports` or `re-uses existing reports`. This isn't something I request changes for though, just for next time.